mne_bids/tests/test_write.py

```diff
@@ -386,7 +386,15 @@ def test_make_dataset_description(tmp_path, monkeypatch):
 
     with open(op.join(tmp_path, "dataset_description.json"), encoding="utf-8") as fid:
         dataset_description_json = json.load(fid)
-        assert dataset_description_json["Authors"] == ["MNE B.", "MNE P."]
+        assert dataset_description_json["Authors"] == ["MNE B.", "MNE P.", "MNE Ł."]
+
+    # If the text on disk is unicode, json.load will convert it. So let's test that the
+    # text was encoded correctly on disk.
+    with open(op.join(tmp_path, "dataset_description.json"), encoding="utf-8") as fid:
```

Comment:
Instead of closing and re-opening the file, you can rewind the "playhead" to the start of the open file, then use `fid.read()` as usual

```suggestion
        # If the text on disk is unicode, json.load will convert it. So let's test that
        # the text was encoded correctly on disk.
        fid.seek(0)
```